logger.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015 SUSE LLC. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+)
+
+const (
+	// The default path for the logger if nothing has been specified.
+	defaultLogPath = "suseconnect.log"
```

Comment:
`/var/log` is owned by the `root` user. So, I thought that by default we could just choose the current directory since it would be easier for the user to set it up. But I can change it if you want ;)